src/script/descriptor.cpp

```diff
@@ -786,6 +787,11 @@ class DescriptorImpl : public Descriptor
             arg->GetPubKeys(pubkeys, ext_pubs);
         }
     }
+||||||| parent of 657176b834d (descriptors: Add DescriptorImpl::Clone)
+=======
+
+    virtual std::unique_ptr<DescriptorImpl> Clone() const = 0;
+>>>>>>> 657176b834d (descriptors: Add DescriptorImpl::Clone)
```

Comment:
Rebase issues in ed00e60c8e8 that are removed on 06caba093a.

Reply:
Fixed